### DISCUSSION TITLE
WhereNotIn and $get

### DISCUSSION THREAD

**Discussion by warsemann** [filamentphp/filament#5946] [Help] [answered]
Hi, how can i pass $get to closure with WherNotIn clause ?

Select::make('establishment_id')->options(

     Establishment::whereNotIn('id', function($query) use ($get){
                  $query->select('establishment_id')->from('workflows_settings')->where('model_type', '=', $get('model_type'))- 
                  >pluck('establishment_id','establishment_id');
      })
))->pluck('name', 'id'))

**Comment by danharrin** [comment, marked as answer]
`options(fn ($get) => ` and then add the query you posted

**Reply by warsemann** [reply]
Thanks !
